app/models/authorization.rb

```diff
@@ -20,38 +21,48 @@ def persisted?
   end
 
   def cache_key
-    token_key = OpenSSL::Digest::MD5.hexdigest([token.scopes, token.resources].map(&:to_s).join(""))
+    token_key =
+      if api_admin
+        "api-admin"
```

Comment:
is this value used elsewhere? should it be a constant?

Reply:
Only used here, but yeah, probably good practice to make it a constant.  Will update!